### PR TITLE
Groups: Remove group analytics feature flags

### PR DESCRIPTION
**Pull request by macobo**
## Changes

To be merged just before the release.

Decided on this instead of PERSISTENT_FEATURE_FLAGS because we're releasing the feature to all.

Will be merged once we're ready for release, this includes:
- [ ] Blog post
- [ ] Docs
- [ ] Upsell support finish
- [ ] Last touches / fixes

See https://github.com/orgs/PostHog/projects/19/views/1 for more details

### CODE REVIEW

File: frontend/src/lib/introductions/groupsAccessLogic.ts [hunk 1]
```diff
@@ -21,34 +19,25 @@ export const groupsAccessLogic = kea<groupsAccessLogicType<GroupsAccessStatus>>(
...
     selectors: {
-        groupsCanBeEnabled: [
-            (s) => [s.featureFlags, s.clickhouseEnabled],
-            (featureFlags, clickhouseEnabled) => featureFlags[FEATURE_FLAGS.GROUP_ANALYTICS] && clickhouseEnabled,
-        ],
+        groupsCanBeEnabled: [(s) => [s.clickhouseEnabled], (clickhouseEnabled) => clickhouseEnabled],
         groupsEnabled: [
             (s) => [s.groupsCanBeEnabled, s.hasAvailableFeature],
             (groupsCanBeEnabled, hasAvailableFeature) =>
                 groupsCanBeEnabled && hasAvailableFeature(AvailableFeature.CORRELATION_ANALYSIS),
```
Comment (neilkakkar) [inline frontend/src/lib/introductions/groupsAccessLogic.ts:33]: GROUPS feature?

Comment (macobo) [reply frontend/src/lib/introductions/groupsAccessLogic.ts:33]: Wait, wtf :D Will fix asap.

Comment (macobo) [reply frontend/src/lib/introductions/groupsAccessLogic.ts:33]: https://github.com/PostHog/posthog/pull/7638

File: frontend/src/lib/constants.tsx [hunk 2]
```diff
@@ -80,8 +80,6 @@ export const FEATURE_FLAGS = {
     NEW_SESSIONS_PLAYER: 'new-sessions-player', // owner: @rcmarron
     NEW_SESSIONS_PLAYER_EVENTS_LIST: 'new-sessions-player-events-list', // owner: @rcmarron / @alexkim205
     BREAKDOWN_BY_MULTIPLE_PROPERTIES: '938-breakdown-by-multiple-properties', // owner: @pauldambra
-    GROUP_ANALYTICS: 'group-analytics', // owner: @macobo
-    GROUP_ANALYTICS_INTRODUCTION: 'group-analytics-introduction', // owner: @macobo
```
Comment (paolodamico) [inline frontend/src/lib/constants.tsx:84]: I would keep this FF (upsell) instead and use the persisted flags, we may want to do adjustments on Cloud.

**Review by paolodamico — APPROVED** [review]
one suggestion but lgtm! code freeze is on Monday but I think we can already merge this?

**Comment by macobo** [comment]
Please don't - it's intentional we're removing the upsell flag and want to make sure all parts are in before merging this.

File: frontend/src/lib/constants.tsx (hunk 2 above)
Comment (macobo) [reply frontend/src/lib/constants.tsx:84]: We discussed this with Marcus, didn't want to keep noisy flags around.

**Pull request merged**